test/common/testlib.py

```diff
@@ -729,7 +734,7 @@ def run(self, result=None):
         max_retry_hard_limit = 10
         for retry in range(0, max_retry_hard_limit):
             try:
-                super(MachineCase, self).run(result)
+                super(BaseCase, self).run(result)
```

Comment:
Let's please avoid redundancy and possible errors by just using `super()`. We can rely on py3 now.

Reply:
ack